_posts/blog/2018-10-11-review-of-periodicos-e-literatura.md

```diff
@@ -0,0 +1,27 @@
+
+The selection of newspapers organize the collection but the materials also include manuscripts, maps, photographs, drawings and transcriptions of literary works. The [*sixty-six*](http://bndigital.bn.gov.br/dossies/periodicos-literatura/titulos-periodicos-literatura/indice-dos-titulos/) newspapers are organized alphabetically by title, span from the early 1800s to the beginning of the 1900s and each entry includes a brief monograph detailing: the form of the actual newspaper, the editorial aims and significant writers that participated as collaborators. The [*authors*](http://bndigital.bn.gov.br/dossies/periodicos-literatura/personagens-periodicos-literatura/indice-dos-personagens/) have a separate section with a biographic sketch, a transcription of an excerpt of a text published in the newspaper and a link to the digital object of the physical book, if available, and where the text was published. The nine items in the [*manuscripts section*](http://bndigital.bn.gov.br/dossies/periodicos-literatura/titulos-manuscritos/apresentacao-titulos-manuscritos/) are handwritten newspapers and typesetting sketches with their own descriptions for each. The sixteen [*publishing houses*](http://bndigital.bn.gov.br/dossies/periodicos-literatura/tipografias/tipografias-cariocas/) also have individual pages describing their historical location with its contemporary address and include hyperlinks to the newspapers in which they published as well as a historical map of the area. The last two sections list the affiliated [*projects*](http://bndigital.bn.gov.br/dossies/periodicos-literatura/pesquisas-e-pesquisadores/pesquisas/), [*researchers*](http://bndigital.bn.gov.br/dossies/periodicos-literatura/pesquisas-e-pesquisadores/pesquisadores/) and [*publications*](http://bndigital.bn.gov.br/dossies/periodicos-literatura/publicacoes/) that were informed by this project or that were used to generate the content for the descriptions. 
+
+In order to understand the collection, it's important to understand the organizing principles of the larger news archive from which the materials have been selected. This is in part because the coordinators of the project are part of the staff of the NDLB. The Hemeroteca's role is also important because the NDLB was conceived as an extension of the Foundation; differing only in that it is also responsible for uniting and completing collections housed in different institutions and facilitating digital collection development for partners. The constitutive goals of [*preserving national memory and activating its other collections*](http://bndigital.bn.gov.br/sobre-a-bndigital/missao/) is explicitly privileged as well in this project. However, the introduction of the digital is framed ambivalently and this results in issues discussed later. 
```

Comment:
I'm not sure what "the introduction of the digital" means here. Even if you want to withhold a deeper discussion of it until later, some clarification of the term would help.